server/graphql/types/station.js

```diff
@@ -47,15 +62,21 @@ export const resolvers = {
           return trip.route_id
         )
 
-        for route in routes
-        filter route.route_id in routes_of_stops
-        limit ${from}, ${length}
-        return route
-      `).then(cursor => cursor.all())
-    }
-  }
+        let routes = (
+          for route in routes
+          filter route.route_id in routes_of_stops
+          limit ${from}, ${length}
+          return route
+        )
+
+      return routes
```

Comment:
Oui en fait AQL n'a que faire de l'indentation. Le `for` va jusqu'au `return` donc c'est OK.